Add migrate script

naoya-maeda opened the pull request:
To cope with the change in specifications due to [this](https://github.com/simonoliver/UnityFigmaBridge/pull/25) PR, 
I have implemented a mechanism to perform the migration process.

I think that there are other ways to implement this migration and I would like to get your opinion.

simonoliver commented:
Thanks for this - I had a think about migration strategies. I think we will need them at some point in the future but for now, I've just put this to checking existing asset formats and added your code to this commit (so won't accept this PR).

https://github.com/simonoliver/UnityFigmaBridge/commit/87a9e0a50964dcfae279bb8d93574e130b68e12c

I've also added in a fix for incorrect alpha blending (the dock and title bar were not dark enough in linear). Let me know if this is working for you ok.

If you are happy with this branch, I'll merge into main. Thanks again

naoya-maeda commented:
Thank you for your response.

> Thanks for this - I had a think about migration strategies. I think we will need them at some point in the future but for now, I've just put this to checking existing asset formats and added your code to this commit (so won't accept this PR).

It looks good. I was not very confident about this PR.
I will now close this PR.

> I've also added in a fix for incorrect alpha blending (the dock and title bar were not dark enough in linear). Let me know if this is working for you ok.
> If you are happy with this branch, I'll merge into main. Thanks again

Great!
I was not recognized that the alpha channel is also affected by the color space setting.
It seems to work fine.
